metagpt/config.py

```diff
@@ -49,11 +49,11 @@ def __init__(self, yaml_file=default_yaml_file):
 
         self.openai_api_base = self._get("OPENAI_API_BASE")
         if not self.openai_api_base or "YOUR_API_BASE" == self.openai_api_base:
```

Comment:
It may work to update the code from 
```python
        ...
        if not self.openai_api_base or "YOUR_API_BASE" == self.openai_api_base:
            openai_proxy = self._get("OPENAI_PROXY") or self.global_proxy
            if openai_proxy:
                openai.proxy = openai_proxy
            else:
                logger.info("Set OPENAI_API_BASE in case of network issues")
        ...
```
to
```python
        ...
        openai_proxy = self._get("OPENAI_PROXY") or self.global_proxy
        if openai_proxy:
            openai.proxy = openai_proxy
        if (not self.openai_api_base or "YOUR_API_BASE" == self.openai_api_base) and not openai_proxy:
            logger.info("Set OPENAI_API_BASE/OPENAI_PROXY in case of network issues")
        ...
```